lldb/test/Shell/SymbolFile/DWARF/ignored_artificial_fields.test

```diff
@@ -0,0 +1,17 @@
+# UNSUPPORTED: system-darwin, system-windows
```

Comment:
Why unsupported on Darwin? Pretty sure it should just work

Reply:
I think we can just remove these and if the test does end up failing we'll just add the requirements back after the fact.

Reply:
Oh you're using gcc for this. Nvm